web/html/src/components/input/FormGroup.tsx

```diff
@@ -3,8 +3,16 @@ import * as React from "react";
 type Props = {
   isError: boolean;
   children: React.ReactNode;
+
+  /** CSS class name to apply to the component */
+  className?: string;
 };
 
 export function FormGroup(props: Props) {
-  return <div className={`form-group${props.isError ? " has-error" : ""}`}>{props.children}</div>;
+  const className = props.className != null ? props.className : '';
+  return <div className={`form-group${props.isError ? " has-error" : ""} ${className}`}>{props.children}</div>;
+}
+
+FormGroup.defaultProps = {
+  className: undefined,
```

Comment:
If the default value is `undefined` there's no need to add this in my opinion, the default value of everything that doesn't exist is `undefined`.

Reply:
Right, IIRC flow wasn't always happy without those, but in TS it seems OK: makes more sense to me at least.